rdr_service/genomic/genomic_job_controller.py

```diff
@@ -494,6 +495,7 @@ def generate_manifest(self, manifest_type, _genome_type, **kwargs):
                     recordCount=result['record_count'],
                     rdrProcessingComplete=1,
                     rdrProcessingCompleteDate=now_time,
+                    fileName=new_file_path.split('/')[2]
```

Comment:
We may want to change this to `new_file_path.split('/')[-1]` in case there is another sub-folder in the path. That way it always pulls the file name.